src/Pages/Home/Showcase/Map.jsx

```diff
@@ -1,8 +1,27 @@
-import { Box } from "@chakra-ui/react";
+import { Box, Image } from "@chakra-ui/react";
 import React from "react";
 
 const Map = () => {
-  return <Box>Map</Box>;
+  return (
+    <Box
+      mt={{
+        sm: "40px",
+        md: "40px",
+        lg: "40px",
+      }}
```

Comment:
This is unnecessary if all the values are the same then you don't need the responsiveness